cf_remote/commands.py

```diff
@@ -296,7 +296,7 @@ def _iterate_over_packages(tags=None, version=None, edition=None, download=False
     else:
         for artifact in artifacts:
             if download:
-                download_package(artifact.url)
+                download_package(artifact.url, directory=output_dir)
```

Comment:
Take a look at the ticket again. We want to keep downloading to the same place, and then copy once it's done downloading and also checksum is verified.